gluten-core/src/main/java/org/apache/gluten/jni/JniWorkspace.java

```diff
@@ -99,12 +85,17 @@ public static void enableDebug(String debugDir) {
     }
   }
 
-  public static JniWorkspace getDefault() {
+  public static void initializeDefault(String rootDir) {
     synchronized (DEFAULT_INSTANCE_INIT_LOCK) {
```

Comment:
Do we still need `synchronized`? Seems it can only be executed by one thread in `initialize` context after this refactoring.

Reply:
There is also `synchronized` used in other function, I will remove them all in another pr.

Reply:
In tests, driver and executors are in the same process, so `initialize` may be call by driver and executor at the same time.